lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -401,6 +401,7 @@ class MailboxDashBoardController extends ReloadableController with UserSettingPo
     } else if (success is GetAllIdentitiesSuccess) {
       _handleGetAllIdentitiesSuccess(success);
     } else if (success is GetComposerCacheSuccess) {
+      _removeComposerCacheOnWeb();
```

Comment:
Why we need to remove cache here?

Reply:
IMO, we should delete it when successfully reopened from cache so that when the user re-enters the url from the browser address bar, the composer can be closed. Of course, the cases of `reload`, or `badCredential` still save the composer to cache.


https://github.com/user-attachments/assets/b1f25fd4-7e16-4dec-8d54-f8f936fba7e4